src/Commands/BuildCommand.php

```diff
@@ -7,37 +7,59 @@
 use Illuminate\Support\Str;
 use Native\Electron\Concerns\LocatesPhpBinary;
 use Native\Electron\Facades\Updater;
-
-use function Laravel\Prompts\select;
+use Native\Electron\Traits\OsAndArch;
 
 class BuildCommand extends Command
 {
     use LocatesPhpBinary;
+    use OsAndArch;
+
+    protected $signature = 'native:build
+        {os? : The operating system to build for (all, linux, mac, win)}
+        {arch? : The Processor Architecture to build for (x64, x86, arm64)}
+        {--publish : to publish the app}';
 
-    protected $signature = 'native:build {os? : The operating system to build for (all, linux, mac, win)}';
+    protected $availOs = ['win', 'linux', 'mac', 'all'];
```

Comment:
Can we name this better than `$availOs` - prefer full words over abbreviations

Reply:
I've changed variable and function names accordingly